.github/workflows/windows_release.yml

```diff
@@ -57,8 +57,8 @@ jobs:
         set -x
         echo "Downloading MiKTeX CLI installer"
         # We download from a specific miror already # TODO: Should store this setup package somewhere ourselves
-        curl -L -O https://ctan.math.illinois.edu/systems/win32/miktex/setup/windows-x64/miktexsetup-4.1-x64.zip
-        unzip miktexsetup-4.1-x64.zip
+        curl -L -O https://ctan.math.illinois.edu/systems/win32/miktex/setup/windows-x64/miktexsetup-4.2-x64.zip
+        unzip miktexsetup-4.2-x64.zip
```

Comment:
Not sure why this was changed but it looks harmless.

Reply:
The windows build was failing, because miktexsetup-4.1 is no longer on the host site. It's 4.2 now. The Mac release is failing for a similar reason now (looking for an older version of miktex) but I can't see how to fix that one.